### ISSUE TITLE
Node-gyp projects not building on Windows after installing 1.5.2.1

### ISSUE THREAD

**Issue by JoshuaColeman** [meteor/meteor#9153]
**Affected Meteor Version:** 1.5.2.1 
**Previously working Meteor Version:** 1.5.1
**Platform:** Windows (observed on Server 2008 R2 and Server 2016)

**Expected behavior:** The meteor toolchain (npm/node-gyp) can be used to build node native modules. For example, the async_pi_estimate module from: https://github.com/nodejs/nan/tree/master/examples/async_pi_estimate

**Actual behavior:** There is a critical gyp failure where 'common.gypi' is not being found.
**Console logs:** 
> C:\nan\examples\async_pi_estimate>meteor npm install
> 
> > async_work@0.0.0 install C:\async_pi_estimate
> > node-gyp rebuild
> 
> gyp: C:\Users\Administrator\AppData\Local\.meteor\packages\meteor-tool\1.5.2_1\mt-os.windows.x86_32\dev_bundle\common.gypi not found (cwd: C:\async_pi_estimate) while reading includes of binding.gyp while trying to load binding.gyp
> gyp ERR! configure error
> gyp ERR! stack Error: `gyp` failed with exit code: 1
> gyp ERR! stack     at ChildProcess.onCpExit (C:\Users\Administrator\AppData\Local\.meteor\packages\meteor-tool\1.5.2_1\mt-os.windows.x86_32\dev_bundle\lib\node_modules\node-gyp\lib\configure.js:336:16)
> gyp ERR! stack     at emitTwo (events.js:87:13)
> gyp ERR! stack     at ChildProcess.emit (events.js:172:7)
> gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:211:12)
> gyp ERR! System Windows_NT 10.0.14393
> gyp ERR! command "C:\\Users\\Administrator\\AppData\\Local\\.meteor\\packages\\meteor-tool\\1.5.2_1\\mt-os.windows.x86_32\\dev_bundle\\bin\\node.exe" "C:\\Users\\Administrator\\AppData\\Local\\.meteor\\packages\\meteor-tool\\1.5.2_1\\mt-os.windows.x86_32\\dev_bundle\\lib\\node_modules\\node-gyp\\bin\\node-gyp.js" "rebuild"
> gyp ERR! cwd C:\async_pi_estimate
> gyp ERR! node -v v4.8.4
> gyp ERR! node-gyp -v v3.6.0
> gyp ERR! not ok
> npm ERR! code ELIFECYCLE
> npm ERR! errno 1
> npm ERR! async_work@0.0.0 install: `node-gyp rebuild`
> npm ERR! Exit status 1
> npm ERR!
> npm ERR! Failed at the async_work@0.0.0 install script.
> npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
> 
> npm ERR! A complete log of this run can be found in:
> npm ERR!     C:\Users\Administrator\AppData\Local\.meteor\packages\meteor-tool\1.5.2_1\mt-os.windows.x86_32\dev_bundle\.npm\_logs\2017-09-28T22_57_52_287Z-debug.log

**Reproduction Steps:**
1. Start with a clean(ish) Windows environment (Tested on Server 2008 R2 and Server 2016). Uninstall Meteor from Add/Remove Programs at a minimum.
2. Download and install Meteor from https://install.meteor.com/windows
3. Open a command prompt and run the command:
`meteor npm install --global windows-build-tools`
4. Clone the nan repo from: https://github.com/nodejs/nan
5. Open a command prompt and attempt to build the example native module from nan by running:
`C:\nan\examples\async_pi_estimate>meteor npm install`

**Discussion:**
I can not find a way to install the old version of meteor (1.5.1) on windows to make this work. I have also tried `meteor --release 1.5.1 npm install` but that does not work. Ideas?

**Comment by dossorio** [comment]
I think I solve that problem by running:
- `meteor remove accounts-password`
- `meteor add accounts-password`

But I tried so many things that I am not completely sure now, hopefully it helps !

**Comment by hwillson** [comment]
@JoshuaColeman I'm curious if the 1.5.2.2 RC will fix this for you. Can you try

```
meteor update --release 1.5.2.2-rc.0
```

and let us know? Thanks!

**Comment by JoshuaColeman** [comment]
@hwillson I installed 1.5.2.2-rc.0 but am still seeing the error. The gyp error is still refering to the 1.5.1.2 meteor tool paths (?). 

I will test upgrading directly from 1.5.1->1.5.2.2-rc.0 and update with the results.

**Comment by JoshuaColeman** [comment]
Is there a way to use the alternate windows installation instructions to install Meteor 1.5.1 explicitly?

**Comment by benjamn** [comment]
Meteor 1.5.2.2-rc.1 includes a fix specifically for this problem. Try it please!

**Comment by JoshuaColeman** [comment]
Awesome! Just tried updating to 1.5.2.2-rc.1 but got this error:
```
C:\Users\Administrator>meteor update --release 1.5.2.2-rc.1
This project uses Meteor 1.5.2.2-rc.1, which isn't available on Windows. ...
```

**Comment by JoshuaColeman** [comment]
Confirmed fixed in 1.5.2.2-rc.1! Thanks!

**Comment by vivekr2212** [comment]
I have currently download meteor version 1.6.0.1 using choco install meteor and installmeteor.exe both..  When i run my project having version 1.5.2.2, it starts showing like
 "Downloading meteor-tool@1.5.2.2..."
 and after some time it displays as
 "Sorry, this project uses Meteor 1.5.2.2, which is not installed and could not be downloaded. Please check to make sure that you are online".
I tried most but gets same above error.
Please anyone help me b'coz of urgent work....please.
